src/AutopilotApi/Client.php

```diff
@@ -54,7 +54,9 @@ public function activate(string $site_id)
             'json' => [
                 'id' => '',
                 'workspaceId' => '',
-                'settings' => (object) [],
+                'settings' => [
```

Comment:
it was previously casting it to an object; is that not needed anymore? My guess is not because both an object and a key-indexed value will produce the same json; just double-checking.

Reply:
No need in explicit casting, yes